packedForest/src/forestTypes/unsupervisedForests/stratifiedInNodeClassIndicesUnsupervised.h

```diff
@@ -66,30 +63,45 @@ namespace fp{
 
 				for(int n=0; n<numUnusedObs; ++n){
 					outSamples[fpSingleton::getSingleton().returnLabel(potentialSamples[randomObsID])].push_back(potentialSamples[n]);
-				
 					outSamps.push_back(potentialSamples[n]);
 					}
 				
 			}
 
+			inline void createInAndOutSetsBagging(const int &numObs, float bagging){
+                                //TODO: We might want to refactor this when we move this over to the binned version.
```

Comment:
somehow we seem to have a mix of tabs and spaces. i don't care which we use -- the C++ was mostly tabs as James wrote it, but we should stick to one or the other -- otherwise things are formatted weird